specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1138,7 +1138,7 @@ Connector architecture.
 The JDBC 4.2 specification is available at
 _https://jcp.org/en/jsr/detail?id=221_ .
 
-===== [[a2553]]Jakarta™ API for XML Web Services (JAX-WS) Requirements
+===== [[a2553]]Jakarta XML Web Services (JAX-WS) 2.3 Requirements
```

Comment:
Each chapter should (TM) the Jakarta on the first use.  If this is the first use in the API.adoc file, then we should use the (TM).  If the first use already has the (TM), then we're good.

Reply:
My bad... thought you meant once per document.

Reply:
I would think once per document is fine unless you're publishing the chapters independently.

Reply:
Just to play it safe, I've been advocating for the first use in each chapter.

Reply:
Seems excessive, but ok.